networking_aci/plugins/ml2/drivers/mech_aci/driver.py

```diff
@@ -265,6 +272,26 @@ def delete_subnet_postcommit(self, context):
         self.rpc_notifier.delete_subnet(context.current, external=external, address_scope_name=address_scope_name,
                                         last_on_network=last_on_network)
 
+    @registry.receives(aci_const.CC_FABRIC_TRANSIT, [events.AFTER_CREATE])
+    def on_fabric_transit_created(self, resource, event, trigger, payload):
+        network_id = payload.metadata['network_id']
+        host = payload.metadata['host']
+        LOG.info("Got transit creation notification for transit %s network %s, syncing network",
+                 network_id, host)
+
+        # get network sync data
+        try:
+            network = context._plugin.get_network(context._plugin_context, network_id)
+        except n_exc.NetworkNotFound as e:
+            LOG.error("Could not sync transit %s network %s - network does not exist! (Error was %s)",
+                      host, network_id, e)
+            return
+
+        sync_data = self.rpc_api._get_network([network])
```

Comment:
Why does `_get_network()` get a list here? It doesn't look like it should take one.
> networking_aci/plugins/ml2/drivers/mech_aci/rpc_api.py:    def _get_network(self, network):

variable is named "network" not "networks"